### DISCUSSION TITLE
Search

### DISCUSSION THREAD

**Discussion by MaksGill** [outline/outline#4902] [General]
Good afternoon. Guys, can you tell me if there is an opportunity to do a Wiki search not by collections, but directly by sections? Or choose some kind of search filter, only by specific sections for more detail?

**Comment by thumDer** [comment]
We also feel that it might be sometimes necessary to filter down the results to pages under a specific page, but afaik it is not yet possible. Could it be a feature request?
Like in confluence:
![kép](https://user-images.githubusercontent.com/54556708/229169662-de7ddd8c-2066-416d-9c4b-4794a30dcbd1.png)
